sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/throughputControl/controller/request/IThroughputRequestController.java

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation.throughputControl.controller.request;
+
+import com.azure.cosmos.implementation.throughputControl.controller.IThroughputController;
+import reactor.core.publisher.Mono;
+
+public interface IThroughputRequestController extends IThroughputController {
+    Mono<Void> renewThroughputUsageCycle(double throughput);
```

Comment:
please add documentation to the interface method what the expectation from the implementation is.

Reply:
added